src/_guides/libraries/library-tour.md

```diff
@@ -1035,6 +1035,38 @@ For more information, see
 [Exceptions](/guides/language/language-tour#exceptions)
 (in the language tour) and the [Exception API reference.][Exception]
 
+### Weak references and finalizers
+Dart is a [garbage-collected][] language,
+which means that any Dart object
+that isn't referenced by at least one other object
+will eventually be disposed by the garbage collector. 
```

Comment:
"will eventually be" -> "can be"

(We can dive into a deep hole of definitions here. Integers are objects, but small integers are never disposed. Neither are Strings which happen to be const in the program.)